config/settings.yml

```diff
@@ -78,6 +78,11 @@
   :use_vim_broker: true
 :container_scanning:
   :concurrent_per_ems: 3
+  :by_provider_name:
+    :_default:
```

Comment:
@enoodle I already mentioned on #10648 why I dislike this approach. cc @blomquisg @Fryguy